Modules/OptionHolder.cs

```diff
@@ -2997,19 +3000,34 @@ public static void Load()
         DisableAirshipVital = BooleanOptionItem.Create(22919, "DisableAirshipVital", false, TabGroup.GameSettings, false)
             .SetParent(DisableAirshipDevices)
             .SetGameMode(CustomGameMode.Standard);
-        DisableDevicesIgnoreConditions = BooleanOptionItem.Create(22920, "IgnoreConditions", false, TabGroup.GameSettings, false)
+        DisableFungleDevices = BooleanOptionItem.Create(22920, "DisableFungleDevices", false, TabGroup.GameSettings, false)
             .SetParent(DisableDevices)
             .SetGameMode(CustomGameMode.Standard);
-        DisableDevicesIgnoreImpostors = BooleanOptionItem.Create(22921, "IgnoreImpostors", false, TabGroup.GameSettings, false)
+        DisableFungleCamera = BooleanOptionItem.Create(22921, "DisableFungleCamera", false, TabGroup.GameSettings, false)
+            .SetParent(DisableFungleDevices)
+            .SetGameMode(CustomGameMode.Standard);
+        DisableFungleVital = BooleanOptionItem.Create(22922, "DisableFungleVital", false, TabGroup.GameSettings, false)
+            .SetParent(DisableFungleDevices)
+            .SetGameMode(CustomGameMode.Standard);
+        DisableDevicesIgnoreConditions = BooleanOptionItem.Create(22923, "IgnoreConditions", false, TabGroup.GameSettings, false)
+            .SetParent(DisableDevices)
+            .SetGameMode(CustomGameMode.Standard);
+        DisableDevicesIgnoreImpostors = BooleanOptionItem.Create(22924, "IgnoreImpostors", false, TabGroup.GameSettings, false)
+            .SetParent(DisableFungleDevices)
```

Comment:
Seems the settings are located somehow randomly and incorrectly

Reply:
Yeah sorry if this seems wrong I kinda fixed it on v1.1.1 and since a lot of things changed in 1.2.0 there is maybe some placements of code
My bad

Reply:
I fixed it